torch/jit/__init__.py

```diff
@@ -28,11 +28,22 @@
 import warnings
 import weakref
 
+from typing import TypeVar, Generic
+
+# See https://github.com/python/typing/issues/449
+try:
+    # Python 3.6 and earlier
+    from typing import GenericMeta
+except ImportError:
+    GenericMeta = type
+
 
 # These are imported so users can access them from the `torch.jit` module
 from torch._jit_internal import Final, _overload, _overload_method
 from torch._jit_internal import ignore, export, unused
 
+T_co = TypeVar('T_co', covariant=True)
```

Comment:
This isn't used in this file, so could use a comment about why it needs to be here.

Reply:
Dead code, should be deleted